flask/dbsetup.py

```diff
@@ -7,30 +7,14 @@
 c.execute("DROP TABLE IF EXISTS posts")
 
 c.execute("""CREATE TABLE posts (
-	alert text not null,
-	alert_end text not null,
-	alert_end_date text not null,
-	alert_end_time text not null,
-	alert_start text not null,
-	alert_start_date text not null,
-	alert_start_time text not null,
-	audience_0 text not null,
-	audience_1 text not null,
-	audience_2 text not null,
 	category_id text not null,
 	category_name text not null,
 	created_at text not null,
-	editable text not null,
 	email text not null,
-	expiration_date text not null,
 	fullname text not null,
 	ID text primary key,
-	location_0 text not null,
-	location_1 text not null,
 	message text not null,
-	owner text not null,
```

Comment:
Weren't we going to keep the audience so that we can differentiate the data for each type of user?